app/scripts/modules/dcos/loadBalancer/configure/wizard/ports.controller.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+const angular = require('angular');
+
+module.exports = angular.module('spinnaker.loadBalancer.configure.dcos.ports', [])
```

Comment:
Packages should start with `spinnaker.dcos`, e.g. `spinnaker.dcos.loadBalancer.configure.ports`

(not sure how consistent we've been throughout the codebase, but something we're trying to enforce now)